plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConnectorTest.java

```diff
@@ -2940,6 +2940,13 @@ public void testInsertUnicode()
         testWithAllStorageFormats(this::testInsertUnicode);
     }
 
+    @Test
+    @Override
+    public void testInsertHighestUnicode()
+    {
+        throw new SkipException("Covered by testInsertUnicode");
```

Comment:
move it above `testInsertUnicode`, so that the two `testInsertUnicode` overloads remain next to each other